packages/webview_flutter/webview_flutter_wkwebview/darwin/webview_flutter_wkwebview/Sources/webview_flutter_wkwebview/WebKitLibrary.g.swift

```diff
@@ -5978,28 +5959,9 @@ final class PigeonApiWKUIDelegate: PigeonApiProtocolWKUIDelegate {
       completion(.success(Void()))
       return
     }
-    let pigeonIdentifierArg = pigeonRegistrar.instanceManager.addHostCreatedInstance(
-      pigeonInstance as AnyObject)
-    let binaryMessenger = pigeonRegistrar.binaryMessenger
-    let codec = pigeonRegistrar.codec
-    let channelName: String =
-      "dev.flutter.pigeon.webview_flutter_wkwebview.WKUIDelegate.pigeon_newInstance"
-    let channel = FlutterBasicMessageChannel(
-      name: channelName, binaryMessenger: binaryMessenger, codec: codec)
-    channel.sendMessage([pigeonIdentifierArg] as [Any?]) { response in
-      guard let listResponse = response as? [Any?] else {
-        completion(.failure(createConnectionError(withChannelName: channelName)))
-        return
-      }
-      if listResponse.count > 1 {
-        let code: String = listResponse[0] as! String
-        let message: String? = nilOrValue(listResponse[1])
-        let details: String? = nilOrValue(listResponse[2])
-        completion(.failure(PigeonError(code: code, message: message, details: details)))
-      } else {
-        completion(.success(Void()))
-      }
-    }
+    print(
```

Comment:
This codepath prints an error to the console but never calls `completion`, which seems like a bug. Doesn't the proxy API generator need to call `completion(.failure(...))` in this path?

Reply:
Thats a really good catch. I'll add this fix to https://github.com/flutter/packages/pull/8567. It looks like Kotlin needs this too.